configs/ARM_HYP_verified.cmake

```diff
@@ -21,6 +21,5 @@ set(KernelDangerousCodeInjection OFF CACHE BOOL "")
 set(KernelFastpath ON CACHE BOOL "")
 set(KernelPrinting OFF CACHE BOOL "")
 set(KernelNumDomains 16 CACHE STRING "")
-set(KernelMaxNumBootinfoUntypedCap 166 CACHE STRING "")
 set(KernelRootCNodeSizeBits 19 CACHE STRING "")
 set(KernelMaxNumBootinfoUntypedCaps 50 CACHE STRING "")
```

Comment:
Is there a plan to remove any customization in a follow-up PR, because the defaults we are using seem a good choice and they use the space that is available anyway. Seems "50" is also just a random choice.

Reply:
Mostly just want to keep it stable. I think Kent was right in his [comment on the other PR](https://github.com/seL4/camkes-vm/pull/95#discussion_r1184392672) that this value used to be more important and was picked for a specific board for these verified configs.

When we've made the proofs more generic, I'd be for removing the settings here and just leaving the default, but it'll be a while until that happens.